storage.go

```diff
@@ -2,35 +2,154 @@ package bos
 
 import (
 	"context"
+	"github.com/baidubce/bce-sdk-go/bce"
 	"io"
+	"time"
 
+	"github.com/baidubce/bce-sdk-go/services/bos/api"
+
+	ps "github.com/beyondstorage/go-storage/v4/pairs"
+	"github.com/beyondstorage/go-storage/v4/pkg/iowrap"
+	"github.com/beyondstorage/go-storage/v4/services"
 	. "github.com/beyondstorage/go-storage/v4/types"
 )
 
 func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
-	panic("not implemented")
+	rp := s.getAbsPath(path)
+
+	if opt.HasObjectMode && opt.ObjectMode.IsDir() {
+		if !s.features.VirtualDir {
+			return
+		}
+
+		rp += "/"
+		o = s.newObject(true)
+		o.Mode |= ModeDir
+	} else {
+		o = s.newObject(true)
```

Comment:
We should create this object with `false`, so that `stat` will be called while get a certain object metadata.